### ISSUE TITLE
A11y_AzureDataStudioAAE_PostgreSQl Hyperscale server groups_Deployment Prerequisite_ScreenReader: Screenreader announces wrong information while navigating through column header with help of arrow key.

### ISSUE THREAD

**Issue by sspatil123** [microsoft/azuredatastudio#15753]
https://user-images.githubusercontent.com/85863999/127468222-78990828-6a0b-458b-9886-0bd4d9aa7cd5.mp4

### Actual:
While navigation through 'required tools' table with help of arrow key and focus is on 'Tool' column header. NVDA announces 'Tool clickable ', but it is not clickable element. NVDA announces wrong information.
Similar issue is observed with JAWS.
Similar issue is observed with this 'Azure Sql Managed Instance- Deployment Prerequisite' scenario
### Expected:
While navigation through 'required tools' table with help of arrow key and focus is on 'Tool' column header. NVDA should announce 'Tool' only, NVDA should not announce wrong information.

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#SH_AzureDataStudio_AAE_Jun2021;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#STP;#WCAG1.3.1;#NVDA;#Jaws;#AINotInScope;

### Environment Details:
Application Name: Azure Data Studio-AAE

Windows Version: Win10
Screen reader: NVDA(2020.2), Jaws(2020.2008.24)

### Repro Steps:

1. Launch the application-Azure Data Studio-Insiders and install extension Azure arc and Azure data ClI
2. Connect to Azure Arc Controller
3. Tab till "Connection" and tab to see more button
4. Tab till "New Deployment" and press enter
5. Tab to PostgreSQl Hyperscale server groups and press enter on select button
6. Start NVDA
7. Enter into table and go till tool header with help of arrow keys
8. Check how nvda is announcing the information 

### User Impact:
Screenreader user will get wrong information.

“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**Comment by Charles-Gagnon** [comment]
@alanrenmsft So it looks like this is an issue coming from slickgrid - its attaching a click handler to the header row : https://github.com/kburtram/SlickGrid/blob/master/slick.grid.js#L380

Which then causes screenreaders to announce it as "clickable". 

So we may have to dig into that to see if we can remove the click handler in cases like this where it doesn't seem to be necessary. 

I'll try to look at it at some point, but if you have time to look into it as well that would be great.

**Comment by alanrenmsft** [comment]
@Charles-Gagnon are you able to reproduce the issue? I am not able to make NVDA go to the table header.

**Comment by Charles-Gagnon** [comment]
@alanrenmsft This is in text mode (insert + space). I can repro this. Are you able to take a look at this one? I can probably find some time this week if you're busy with other stuff.

**Comment by krrish2021** [comment]
Closing as not repro
![Screenshot (189)](https://user-images.githubusercontent.com/85863543/150972171-eb8218a0-a535-4c31-aeaa-b2350e33cf4f.png)

**Comment by Harika1702** [comment]
GitHub Tags: #Closed;
